_pages/en_US/troubleshooting.txt

```diff
@@ -162,7 +162,7 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 
 1. Set your DNS settings to "Auto"
 1. Move closer to your WiFi router
-1. Update from Safe Mode by turning off the console, holding L+R+Up+A on boot, and following the prompts
+1. Update from Safe Mode by turning off the console, holding (A) + (Right Shoulder) + (Left Shoulder) + (D-Pad Up) on boot, and following the on screen prompts
```

Comment:
A bit picky, but the guide uses the order of `(Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A)` to describe the buttons.